api-schema/src/main/java/org/stellar/anchor/api/platform/HealthCheck.java

```diff
@@ -0,0 +1,40 @@
+package org.stellar.anchor.api.platform;
+
+import com.google.gson.annotations.SerializedName;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Data;
+
+@Data
+public class HealthCheck {
+  Instant started;
+
+  @SerializedName("elapsed_time_ms")
+  Duration elapsedTime = Duration.ZERO;
+
+  @SerializedName("number_of_checks")
+  int numberOfChecks;
+
+  Map<String, HealthCheckResult> checks = new HashMap<>();
+
+  public HealthCheck() {
+    this.started = Instant.now();
+  }
+
+  public HealthCheck finish(List<HealthCheckResult> results) {
+    for (HealthCheckResult result : results) {
+      checks.put(result.name(), result);
+    }
```

Comment:
👀 if there are keys in `checks` that are not in `results`, we would pass the wrong information that they were checked when they actually weren't.

We should start a new `check` map here instead of simply calling `check.put`.